cmd/podman/images/scp.go

```diff
@@ -203,15 +203,6 @@ func scp(cmd *cobra.Command, args []string) (finalErr error) {
 		}
 	}
 
-	src, err := json.MarshalIndent(source, "", "    ")
-	if err != nil {
-		return err
-	}
-	dst, err := json.MarshalIndent(dest, "", "    ")
-	if err != nil {
-		return err
-	}
-	fmt.Printf("SOURCE: %s\nDEST: %s\n", string(src), string(dst))
```

Comment:
this was meant as a permanent output to tell the user some concrete information about what they transferred and who they transferred to, @rhatdan would you be opposed to removing this? I think having some sort of scp specific output is a good idea.